test.js

```diff
@@ -1392,6 +1392,21 @@ function runTests(options) {
             })();
           });
       });
+      it('should be compatible with the cwd option', function(done) {
+        var spy = sinon.spy();
+        var testPath = getFixturePath('awf_cwd/awf_cwd.txt');
```

Comment:
When introducing a new fixture file to the tests it needs to be added to the `clean` function at the top. Or you could change this to reuse paths that are already on there.

This is likely what caused the tests to explode.